rpcs3/RPCS3App.cpp

```diff
@@ -1,6 +1,7 @@
 #include "RPCS3App.h"
 
 #include "Emu/System.h"
+#include "Utilities/event.h"
```

Comment:
I know I accidentally added this line trying to work with the event system that exited already, but I don't think it's needed anymore.